packages/function-tree/package.json

```diff
@@ -46,8 +46,5 @@
       "**/*.test.js",
       "**/testHelper.js"
     ]
-  },
```

Comment:
The package `mock-socket` is required to run tests in function-tree. Please leave the devDependencies where they are needed. Most of them are missing in other packages and this is wrong and we will need to fix it. I would like to setup a rule where dependencies not in any package's `devDependencies` or `dependencies` will  get removed from main package.json at some point.